cli/npm/resolution/graph.rs

```diff
@@ -61,7 +65,7 @@ enum ResolvedIdPeerDep {
   /// change from under it.
   ParentReference {
     parent: GraphPathNodeOrRoot,
-    child_pkg_nv: NpmPackageNv,
+    child_pkg_nv: Arc<NpmPackageNv>,
```

Comment:
It was expensive to clone these.

Reply:
Does it need to be `Arc`? Would `Rc` work here? I thought we were only constrained to a single thread here

Reply:
Could you open an issue about it and ping me there?

Reply:
#18079